bitset.go

```diff
@@ -520,6 +520,44 @@ func (b *BitSet) DeleteAt(i uint) *BitSet {
 	return b
 }
 
+// AppendTo appends all set bits to buf and returns the (maybe extended) buf.
+//
+// See also [BitSet.AsSlice] and [BitSet.NextSetMany].
+func (b *BitSet) AppendTo(buf []uint) []uint {
```

Comment:
Theoretically, we can overflow uint. It would be nice to mention that what happens if someone has a gigantic bitset.